src/util-debug.h

```diff
@@ -80,8 +80,8 @@ typedef enum {
 #define SC_LOG_DEF_LOG_FORMAT_REL_CONFIG "[%i] %d: %S: %M"
 #define SC_LOG_DEF_LOG_FORMAT_DEBUG      "%d: %S: %M [%n:%f:%l]"
 
-/* The maximum length of the log message */
-#define SC_LOG_MAX_LOG_MSG_LEN 2048
+/* The maximum length of the log message: we add max rule size and other info */
+#define SC_LOG_MAX_LOG_MSG_LEN 8192 + 256
```

Comment:
where does the 256 come from?

Reply:
It was a guess but I've been overly optimistic because if we look at the failure message for a signature, we have in fact inside of the log  message `error parsing signature {{SIG}} from file {{FILE}}`. It we take the full log wihtout the SIG and FILE, we have 265 chars (512 for a round count).
So it we want to be strict we need in fact
```C
#define SC_LOG_MAX_LOG_MSG_LEN 8192 + PATH_MAX + 512
```
PATH_MAX is usually 4092.

Should I switch to that ?